src/natcap/invest/ndr/ndr.py

```diff
@@ -949,7 +949,9 @@ def _map_load_op(lucode_array):
         result = numpy.empty(lucode_array.shape)
         result[:] = _TARGET_NODATA
         for lucode in numpy.unique(lucode_array):
-            if lucode != nodata_landuse:
+            lucode_is_nodata = utils.array_equals_nodata(
+                numpy.array([lucode]), nodata_landuse).any()
+            if not lucode_is_nodata:
                 try:
                     result[lucode_array == lucode] = (
```

Comment:
Can we guarantee `lucode` from `lucode_array` is of integer type? I think we accept floating point LULC rasters... In which case this would be necessary to protect against a rogue `nan` nodata value... This whole thing is certainly not as clean I thought it'd be, which makes me wonder if we should think about it some more.

Reply:
Sorry I think my comment was ambiguous here. I was referring to line 956: by leaving that `==` we're assuming they are ints, but earlier at 952 we're not okay with that assumption, as you point out.